converter/converter.rc

```diff
@@ -0,0 +1,100 @@
+// Microsoft Visual C++ generated resource script.
```

Comment:
Any idea of the purpose of these resource files?

Reply:
these add version info to c++(cli) assemblies, that also show up in file properties->details
AssemblyInfo.cpp is also needed generate AssemblyVersion, rc files don't add it